<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidgetComponents</class>
 <widget class="QDockWidget" name="DockWidgetComponents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>603</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Components</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>353</width>
         <height>582</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="6">
         <widget class="QListWidget" name="listComponents"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="labelComponents">
          <property name="text">
           <string>Component Type:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>comboBoxComponents</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="3">
         <widget class="QComboBox" name="comboBoxComponents">
          <item>
           <property name="text">
            <string>Fuselage</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Wing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Horizontal Tail</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vertical Tail</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Main Landing Gear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nose Landing Gear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Engine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rotor Drive</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Main Rotor Hub</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Main Rotor</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tail Rotor</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>All-Else</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QPushButton" name="pushButtonEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4" colspan="2">
         <widget class="QPushButton" name="pushButtonDel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
